libvast/vast/min_max_synopsis.hpp

```diff
@@ -40,7 +41,40 @@ class min_max_synopsis : public synopsis {
       max_ = *y;
   }
 
-  bool lookup(relational_operator op, data_view rhs) const override {
+  std::optional<bool> lookup(relational_operator op,
+                             data_view rhs) const override {
+    if (op == in) {
+      auto s = caf::get_if<view<set>>(&rhs);
+      if (s == nullptr)
+        return std::nullopt;
+      for (auto element : **s)
+        if (lookup_impl(op, element))
+          return true;
+      return false;
+    } else if (op >= equal && op <= greater_equal) {
+      return {lookup_impl(op, rhs)};
+    } else
```

Comment:
Our style guide says to use either always braces or never, but no partial use of braces in branches.